draft-irtf-cfrg-opaque.md

```diff
@@ -976,16 +976,25 @@ handshake_encrypt_key =
 
 Nh is the output length of the underlying hash function.
 
-The HKDF input parameter `info` is computed as follows:
+The Derive-Secret parameter `info` is computed as:
 
 ~~~
-info = "3DH keys" || I2OSP(len(client_nonce), 2) || client_nonce
-                  || I2OSP(len(server_nonce), 2) || server_nonce
-                  || I2OSP(len(client_identity), 2) || client_identity
-                  || I2OSP(len(server_identity), 2) || server_identity
+info = "3DH" || client_preamble || client_transcript ||
```

Comment:
Can we rename this variable to `preamble` or something, instead of keeping it as `info`? It seems to be called `preamble` in the HMQV/SIGMA-I sections, and `info` is already an overloaded term here, since it could be confused with HKDF info, `client_info`, `enc_server_info `, etc. :)